specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -2611,6 +2732,46 @@
         }
       }
     },
+    "Status": {
+      "description": "The status of the long running operation.",
+      "properties": {
+        "status": {
+          "description": "The status of the long running operation.",
+          "type": "string",
+          "enum": [
+            "running",
+            "succeeded",
+            "failed"
+          ],
+          "x-ms-enum": {
+            "name": "StatusType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "StatusWithLocation": {
+      "description": "The status of the long running operation.",
+      "properties": {
+        "status": {
+          "description": "The status of the long running operation.",
+          "type": "string",
+          "enum": [
+            "running",
+            "succeeded",
+            "failed"
+          ],
+          "x-ms-enum": {
+            "name": "StatusType",
+            "modelAsString": true
+          }
+        },
+        "resourceLocation": {
```

Comment:
consider renaming this to somehing besides "resourceLocation" since a resource's location has a specific meaning in ARM (its region).

Reply:
Updated